gammapy/estimators/map_profile.py

```diff
@@ -0,0 +1,313 @@
+"""Tools to create profiles (i.e. 1D "slices" from 2D images)."""
+import numpy as np
+from astropy import units as u
+from astropy.wcs.utils import proj_plane_pixel_scales
+from astropy.coordinates import Angle, SkyCoord
+from regions import RectangleSkyRegion, PixCoord
+from gammapy.maps import MapAxis
+from gammapy.utils.table import table_from_row_data
+from gammapy.estimators import ImageProfile
+from gammapy.stats import WStatCountsStatistic, CashCountsStatistic
+from gammapy.datasets import SpectrumDatasetOnOff
+
+__all__ = ["MapProfileEstimator", "make_orthogonal_boxes"]
+
+
+class MapProfileEstimator:
+    """Estimate profile from a DataSet.
+
+    Parameters
+    ----------
+    regions : list of `regions`
+        regions to use
+    axis : `~gammapy.maps.MapAxis`
```

Comment:
I think this additional `axis` parameter is confusing somehow, as it duplicates the information, that is already contained in the regions. I think the only solution would be to store the information on the regions in the result table as well....but I'll think about it once more.

Reply:
In a table such that any astropy.regions can be well represented whatever the wcs... damned! If this solution is chosen, I need really some code.